installation/single_machine_rhel.rst

```diff
@@ -75,6 +75,10 @@ Finally, the coordinator needs to know where it can find the workers. To tell it
   psql -p 9700 -c "SELECT * from master_add_node('localhost', 9701);"
   psql -p 9700 -c "SELECT * from master_add_node('localhost', 9702);"
 
+.. note::
```

Comment:
Should these notes be at the end after the "verify installation has succeeded" step?

Reply:
Actually I might want to move the note even earlier, right about where someone is about to start the database. I want to avoid them starting it up, and then learning about the stats collection after it's too late and the software has already reported on them.

Admittedly there's not much for a new empty cluster to report, so it's not that big of a deal. I can move the notice wherever you think best.